docs/index.md

```diff
@@ -0,0 +1,62 @@
+
+# Getting started (napari)
+
+MIDI App Controller is a package designed to integrate MIDI controllers with Python Qt apps using app-model. As of now, it is used most commonly with [napari](napari.org), a viewer for multi-dimensional images. We will show how to use MIDI App Controller with napari but getting started with other applications should look very similar.
+
+## Installation
+
+To install MIDI App Controller in your environment (where Python and napari are already installed), use this command:
+
+(TODO: This doesn't work as the package is not yet published)
+
+```
+pip install midi-app-controller
+```
+
+napari will automatically detect the package and install the plugin next time it starts.
+
+
+### Development version
+
+To install the newest development version, clone the GitHub repo and install it as a local package:
+
+```
+git clone https://github.com/midi-app-controller/midi-app-controller/
+cd midi-app-controller
+python3 -m pip install -e .
+napari
+```
+
+## Setup
+
+Launch the plugin from the _Plugins_ menu.
+
+![](img/plugins-menu.png)
+
+A panel will open to the side.
+
+![](img/midi-status.png)
+
+### Controller
+
+If your MIDI controller is supported out of the box, you can simply select the appropriate model. If not, you will need to tell MIDI App Controller how to interact with this model of controller by creating a [controller schema](controllers.md).
```

Comment:
`by creating a [controller schema]`
But how to do it? There is some info about it in the `Edit binds` section. But I think that a new `.md` file dedicated to config files (read-only files vs user files, default paths, etc.) would be helpful.